index.bs

```diff
@@ -145,36 +149,36 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <h2 id="image-sequences">Image Sequences</h2>
 
-<p>
-    An <dfn>AV1 Image Sequence</dfn> is defined as a set of AV1 [=Temporal Units=] stored in an [=AV1 track=] as defined in [[!AV1-ISOBMFF]] with the following constraints:
-
-    - The [=AV1 Image Item=] should have its <code>[=still_picture=]</code> flag set to 1.
-    - The [=AV1 Image Item=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
-    - The track handler for an [=AV1 Image Sequence=] shall be <code>pict</code>.
-
-    <h2 id="alpha-images">Alpha Image Items and Sequences</h2>
+<p>
+    An <dfn>AV1 Image Sequence</dfn> is defined as a set of AV1 [=Temporal Units=] stored in an [=AV1 track=] as defined in [[!AV1-ISOBMFF]] with the following constraints:
+
+    - The [=AV1 Image Item=] should have its <code>[=still_picture=]</code> flag set to 1.
+    - The [=AV1 Image Item=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+    - The track handler for an [=AV1 Image Sequence=] shall be <code>pict</code>.
+
+    <h2 id="alpha-images">Alpha Image Items and Sequences</h2>
 <p>An <dfn>AV1 Alpha Image Item</dfn> (respectively an <dfn>AV1 Alpha Image Sequence</dfn>) is an [=AV1 Image Item=] (respectively [=AV1 Image Sequence=]</dfn>) with the following additional constraints:
     - The <code>[=mono_chrome=]</code> field in the [=Sequence Header OBU=] shall be set to 1.
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) shall be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-  <h2>Brands and file extensions</h2>
+  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
```

Comment:
The reason I did not add a "id" attribute to the headings is that this is one of the benefits of using Bikeshed instead of authoring directly in HTML. Bikeshed will automatically generate the "id" attribute based on the text in heading, and it can be used for cross-references.

Reply:
It was giving me a warning that they were not defined. If we want to have short, readable anchor names for URLs, it's probably better to define them, e.g. to avoid 'avif' being repeated.